<launch>
  <env name="ROSCONSOLE_CONFIG_FILE"
       value="$(find exploration_alg)/config/rosconsole.conf"/>
  <node pkg="exploration_alg" type="explore_node" name="explore_node" output="screen"/>
</launch>